swarm/api/http/handler.go

```diff
@@ -0,0 +1,120 @@
+package http
+
+import (
+	"encoding/json"
+	"fmt"
+	"io"
+	"net/http"
+	"strings"
+	"time"
+
+	"github.com/ethereum/go-ethereum/swarm/log"
+)
+
+func (s *Server) HandleBzzGet() http.HandlerFunc {
```

Comment:
This could be:
```
func (s *Server) HandleBzzGet(w http.ResponseWriter, r *http.Request) {
...
```

So it does not have to be called in NewServer, just set.

The same stands for all similar functions in this file.